### PR TITLE
Update GmailFile1 - from hotmailplease check

### PR DESCRIPTION
**Pull request by irfansheikh**

### CODE REVIEW

File: GmailFile1 [hunk 1]
```diff
@@ -2,5 +2,7 @@ Class Gmail {
 Public static main()
 {
 print();
+print();
```
Comment (IrfanSheikh2000) [inline GmailFile1:5]: i think this is not right

Comment (irfansheikh) [reply GmailFile1:5]: ok how would you like to change

**Review by IrfanSheikh2000 — DISMISSED** [review]
please add one more print

**Comment by IrfanSheikh2000** [comment]
@irfansheikh  please check again

**Pull request merged**